BDArmory/Modules/ModuleWeapon.cs

```diff
@@ -3200,6 +3174,9 @@ public void RunTrajectorySimulation()
                                 if (simTime > thrustTime)
                                 {
                                     simDeltaTime = 0.1f;
+                                    //var timeOfFlight = BallisticTrajectorySimulation(ref simCurrPos, simVelocity, Mathf.Min(maxTargetingRange, targetDistance), ((simStartPos - simCurrPos).magnitude - maxTargetingRange) / simVelocity.magnitude, simDeltaTime, FlightGlobals.getAltitudeAtPos(targetPosition) < 0);
+                                    //bulletPrediction = simCurrPos;
+                                    //break;
                                 }
 
                                 if (simTime > 0.04f)
```

Comment:
The `0.04f` should probably be `2f * Time.fixedDeltaTime` if it's intended to be 2 frames before the aero-stabilisation kicks in.

Reply:
it's hardcoded in the rocket class to wait 0.04s (instead of the more sensible 2 x fixeddeltaTime)  so that's why. No reason it can't be changed.